docs/products/postgresql/howto/use-pgvector.rst

```diff
@@ -0,0 +1,107 @@
+Enable and use pgvector on Aiven for PostgreSQL®
+================================================
+
+This article provides step-by-step instructions on enabling, using, and disabling the pgvector extension for your Aiven for PostgreSQL service.
+
+About using pgvector
+--------------------
+
+The pgvector extension allows you to perform the vector similarity search and use embedding techniques directly in Aiven for PostgreSQL. See :doc:`pgvector for AI-powered search </docs/products/postgresql/concepts/pgvector>` for more information on what pgvector is and how it works.
+
+Prerequisites
+-------------
+
+* Aiven account
+* Aiven for PostgreSQL service
```

Comment:
We're only releasing it for PG13 and up not sure you need to mention that here but I don't see it mentioned elsewhere.